sdk/appcenter-identity/src/main/java/com/microsoft/appcenter/identity/Identity.java

```diff
@@ -500,7 +502,11 @@ public void onCancel() {
             });
         } else {
             AppCenterLog.debug(LOG_TAG, "signIn is called while it's not configured or not in the foreground, waiting.");
-            mSignInDelayed = true;
+            if (NetworkStateHelper.getSharedInstance(mContext).isNetworkConnected()) {
+                mSignInDelayed = true;
```

Comment:
We are removing background as well and this field altogether.
Background => error.

Reply:
Was discussed with @jaelim-ms and @amchew online but forgot to update the tasks.